examples/native-fungible/README.md

```diff
@@ -0,0 +1,47 @@
+<!-- cargo-rdme start -->
+
+# Native Fungible Token Example Application
+
+This app is very similar to the [Fungible Token Example Application](https://github.com/linera-io/linera-protocol/blob/main/examples/fungible/README.md#fungible-token-example-application). The difference is that this is a native token that will use system API calls for operations.
+The general aspects of how it works can be referred to the linked README.
+
+# How It Works
+
+Refer to [Fungible Token Example Application - How It Works](https://github.com/linera-io/linera-protocol/blob/main/examples/fungible/README.md#how-it-works).
+
+# Usage
+
+## Setting Up
+
+Most of this can also be referred to the [fungible app README](https://github.com/linera-io/linera-protocol/blob/main/examples/fungible/README.md#setting-up), except for at the end when compiling and publishing the bytecode, what you'll need to do will be slightly different:
+
+Compile the `native-fungible` application WebAssembly binaries, and publish them as an application
+bytecode:
+
+```bash
+(cd examples/native-fungible && cargo build --release)
+
+BYTECODE_ID="$(linera publish-bytecode \
+    examples/target/wasm32-unknown-unknown/release/native_fungible_{contract,service}.wasm)"
+```
+
+Here, we stored the new bytecode ID in a variable `BYTECODE_ID` to be reused it later.
+
+## Creating a Token
+
+Most of this can also be referred to the [fungible app README](https://github.com/linera-io/linera-protocol/blob/main/examples/fungible/README.md#creating-a-token), except for at the end when creating the application, you don't need to pass a `ticker_symbol` because for the Native Fungible App that will be hardcoded to `NAT`.
```

Comment:
Actually, you _do_ have to pass a ticker symbol, and it has to be `"NAT"`. (I think the contract asserts that?)

Reply:
We could also clarify that the initial balances are taken from the chain balance, because the app can't mint new native tokens.

Reply:
Right, my bad. Forgot to change it after we started enforcing it. Sounds good